core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Boxable.java

```diff
@@ -18,10 +22,30 @@
 public interface Boxable extends Nameable, Barcodable, SecurableByProfile, Serializable {
 
   public enum EntityType {
-    SAMPLE(SampleImpl.class),
-    LIBRARY(LibraryImpl.class),
-    DILUTION(LibraryDilution.class),
-    POOL(PoolImpl.class);
+    SAMPLE(SampleImpl.class) {
```

Comment:
If you want this to be shorter, you can make the constructor take a `Supplier<? extends AbstractBoxPosition>` and then give `SampleBoxPosition::new` as an argument. Not necessarily worth changing though.

Reply:
:+1: changed